bin.js

```diff
@@ -121,6 +121,14 @@ program
   .option('--host <host>', 'dev server host (default: localhost)')
   .action(wrap('./lib/start'))
 
+program
+  .command('storybook')
+  .description('start the storybook server')
+  .option('--port <port>', 'dev server port (default: 8080)')
+  .option('--host <host>', 'dev server host (default: localhost)')
+  .option('--static', 'Build static version of storybook.')
+  .action(wrap('./lib/storybook'))
+
 program
   .command('disable-idp')
```

Comment:
Remove in a new pr?

Reply:
I think we still need this :man_shrugging: